### PR TITLE
added the force reduction things

### PR DESCRIPTION
**Pull request by ionisea**
changed the shape class so we can detect last frame's acceleration/velocity and added displacement code

### CODE REVIEW

**Review by lashlockbhs — COMMENTED** [review]
run prettier

**Review by lashlockbhs — APPROVED** [review]

**Review by szetingchen — APPROVED** [review]
hes hoping it works

File: projects/physics/public/math.js [hunk 1]
```diff
@@ -83,6 +83,12 @@ const getAcceleration = (force, mass, appliedTime) => {
 const getVelocity = (force, mass, appliedTime, fps) =>
   getAcceleration(force, mass, appliedTime) * (1 / fps);
 
+const getDisplacement = (force, mass, appliedTime, fps, angle) =>{
+  const h = getVelocity(force, mass, appliedTime, fps) * 1/fps
+  const p = Math.cos(90 - angle) * h
```
Comment (gigamonkey) [inline projects/physics/public/math.js:88]: 90? I'm pretty sure Math.cos is expecting radians.

File: projects/physics/public/script.js [hunk 2]
```diff
@@ -25,12 +25,13 @@ import {
...
   getAcceleration,
   getVelocity,
+  getDisplacement,
   vector,
 } from './math.js';
 
 const canvas = document.getElementById('screen');
 setCanvas(canvas);
-
+const FPS = 12 // frames per second
```
Comment (gigamonkey) [inline projects/physics/public/script.js:34]: We should talk about how you're expecting to use this. Ask me about it in class.

**Review by gigamonkey — COMMENTED** [review]
Couple comments.

**Pull request merged**